### ISSUE TITLE
Is there no batch support?

### ISSUE THREAD

**Issue by yippp** [facebookresearch/hgnn#1]
Hi~ I'm very interested in your paper, and I want to run your code. 
I found that in the `BaseTask.py` you set `batch_size=1`, it seems that it should be used for blockchain graph. 
I think larger batch size should accelerate in other database like QM9. But when batch size is large than 1, the error occurs. It seems that due to you have not write code to pad graphs with different size in a batch. Could you tell me is it possible to run with larger batch size?

**Comment by leuchine** [comment]
Hi: Yes. The batch size is set to 1 to consume large blockchain graphs and other large graph datasets. The code does not contain padding yet. We run on multi-GPUs to speed up, where the batch size = num of GPUs. Thanks!

**Comment by yippp** [comment]
Thank you for your prompt reply. I think add larger batch size supply with multi GPUs can achieve higher speed up in some some graph dataset, like QM9.

**Comment by leuchine** [comment]
Yes. You are right. As padding the blockchain graphs is super memory-inefficient, we did not implement the padding part. But padding is possible on small graph datasets like QM9. Thanks!
